src/SlackMessages.ts

```diff
@@ -40,9 +57,9 @@ export function url(fullUrl: string, label?: string) {
  * When userName is provided will add readable user name.
  */
 export function user(userId: string, userName?: string) {
-    if (!emptyString(userId) && !emptyString(userName)) {
+    if (userId && userName) {
```

Comment:
Same here, we don't want to try to convert empty strings into `<|>` or `<>`. Same applies to channel, bold (** will render as 2 starts which is not correct), italic, etc.